app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivity.kt

```diff
@@ -569,25 +570,27 @@ class DeviceShieldTrackerActivity :
             binding.deviceShieldTrackerShieldImage.setImageResource(R.drawable.apptp_shield_disabled)
             binding.deviceShieldTrackerLabelEnabled.gone()
 
-            val (disabledLabel, annotation) = if (runningState.stopReason == REVOKED) {
-                R.string.atp_ActivityRevokedLabel to REPORT_ISSUES_ANNOTATION
-            } else if (runningState.stopReason == SELF_STOP) {
-                R.string.atp_ActivityDisabledLabel to REPORT_ISSUES_ANNOTATION
-            } else {
-                R.string.atp_ActivityDisabledBySystemLabel to RE_ENABLE_ANNOTATION
-            }
-            binding.deviceShieldTrackerLabelDisabled.apply {
-                setClickableLink(
-                    annotation,
-                    getText(disabledLabel),
-                ) {
-                    if (annotation == REPORT_ISSUES_ANNOTATION) {
-                        launchFeedback()
-                    } else if (annotation == RE_ENABLE_ANNOTATION) {
-                        reEnableAppTrackingProtection()
+            if (runningState.stopReason !is SNOOZED) {
```

Comment:
I added this since I don’t think we want the disabled label to be shown for AppTP when snooze has been invoked.